charts/merlin/Chart.yaml

```diff
@@ -33,4 +33,4 @@ maintainers:
 - email: caraml-dev@caraml.dev
   name: caraml-dev
 name: merlin
-version: 0.11.1
+version: 0.11.3
```

Comment:
Somehow `0.11.2` has already been released: https://github.com/caraml-dev/helm-charts/releases/tag/merlin-0.11.2. So I'm bumping it up by 2 levels.

Reply:
Just rebased this branch, so this comment is now invalid 😅